### ISSUE TITLE
Use Aerostack in Gazebo simulator?

### ISSUE THREAD

**Issue by LavrenovRoman** [cvar-upm/aerostack#74]
How to use Aerostack in Gazebo? 
Do you have some examples or launch files for testing Aerostack in a simulation?

**Comment by martin-molina** [comment]
Yes, you can see some examples here: https://github.com/cvar-upm/aerostack/wiki/Example-projects

**Comment by martin-molina** [comment]
You can also see other examples corresponding to Aerostack 5.0 here: https://github.com/aerostack/install/wiki/Example-Applications
